packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -80,6 +81,30 @@ QUnit.module('MessageBox', moduleConfig, () => {
 
             assert.strictEqual(this.$input.val(), emptyValue);
         });
+
```

Comment:
QUnit.test('send button should be disabled after adding a line break into input', function(assert) {
  // Test implementation goes here
});

Reply:
QUnit.test('Send button should be disabled after entering any character and then removing it', function(assert) {
  // Test implementation goes here
});

Reply:
There is a test like this: Send button should be disabled after entering any character and then removing it

Reply:
[332595f30abb7fcb0abcc63677fca5a99e430962]